pro_tes/ga4gh/tes/models.py

```diff
@@ -666,7 +666,7 @@ class TesEndpoint(CustomBaseModel):
     Args:
         host: Host at which the TES API is served that is processing this
             request; note that this should include the path information but
-            *not* the base path path defined in the TES API specification;
+            *not* the base path defined in the TES API specification;
```

Comment:
Make use of all 80 chars in each line of a paragraph. After deleting the repeat `path`, you can move the `e.g.,` to the previous line.

Btw, there are VS Code plugins that can do this automatically (and that you can configure with the desired line length).

Reply:
Done